modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -763,6 +765,63 @@ public abstract class Base${schemaName}ResourceImpl
 				throw new UnsupportedOperationException("This method needs to be implemented");
 			</#if>
 		}
+
+		<#if (privateParsersNeeded?size >= 1)>
+			<#if privateParsersNeeded?seq_contains("boolean")>
+				private Boolean _parseBoolean(String value){
+					Boolean booleanValue = null;
+					if (value != null){
+						booleanValue = Long.parseBoolean(value);
+					}
+
+					return booleanValue;
+				}
+			</#if>
+
+			<#if privateParsersNeeded?seq_contains("date")>
+				private java.util.Date _parseDate(String value){
```

Comment:
Could we import the java.util.Date class and use just Date? maybe it is easier in this case

Reply:
I don't really know. I was doing it becouse before it was done like that but if you think I can do it I will change it.